code/modules/clothing/spacesuits/rig/suits/robotics.dm

```diff
@@ -0,0 +1,29 @@
+//Mining suit
+/obj/item/weapon/rig/robotics
+	name = "advanced suit control belt"
+	suit_type = "advanced suit"
+	desc = "A lightweight suit combining the utility of a RIG with the wearability of a voidsuit."
+	icon_state = "void_explorer2"
+	slot_flags = SLOT_BELT
```

Comment:
This is awesome and I want it right now. That having been said, I need to point out `/mob/living/carbon/human/get_breath_from_internal` currently checks specifically the `human.back` var for finding a rig to breathe out of. You'll need to update that. I notice there's a var on humans called `wearing_rig` that can likely be used, as rigs set it to themselves when they're worn.